pom.xml

```diff
@@ -297,13 +297,6 @@
                 <version>${dep.jjwt.version}</version>
             </dependency>
 
-            <!-- TODO: move to airbase -->
-            <dependency>
-                <groupId>jakarta.servlet</groupId>
-                <artifactId>jakarta.servlet-api</artifactId>
-                <version>6.0.0</version>
```

Comment:
Shouldn't this be part of `Update airbase to 155` commit?

Reply:
it was introduced earlier than 155 as far as I remember

Reply:
Moved to `update to airbase 156` commit